### PR TITLE
Add vsp chart

### PR DESCRIPTION
**Pull request by blairboy362**
Add the chart for the Verify Service Provider to the proxy node.

### CODE REVIEW

File: pki/generate [hunk 1]
```diff
@@ -172,7 +172,7 @@ Dir.chdir(output_dir) do
 
   proxy_node_vars = {
     'HUB_ENTITY_ID' => options.hub_entity_id,
-    'HUB_URL' => "#{options.idp_url}/stub-idp-demo/SAML2/SSO",
+    'HUB_URL' => "#{options.idp_url}/SAML2/SSO",
```
Comment (JHjava) [inline pki/generate]: We might want to change idp_url to hub_url in options.rb going forward.

Comment (vixus0) [reply pki/generate]: We probably shouldn't really be using the script at all...

File: proxy-node-chart/charts/verify-service-provider/templates/config.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  labels:
+    app: {{ template "verify-service-provider.name" . }}
+    chart: {{ template "verify-service-provider.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "verify-service-provider.fullname" . }}
+data:
+  verify-service-provider.yaml: |-
+{{ toYaml .Values.configFile | indent 4 }}
```
Comment (paroxp) [inline proxy-node-chart/charts/verify-service-provider/templates/config.yaml]: New line, please and thank you \<insert troll face\>

File: proxy-node-chart/charts/verify-service-provider/values.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# Default values for verify-service-provider.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: govukverify/verify-service-provider
+  tag: "latest"
```
Comment (paroxp) [inline proxy-node-chart/charts/verify-service-provider/values.yaml]: Not a requirement, but we probably should be pinning that in some way or another.

**Pull request merged**